tests/unit/_experimental/test_plots.py

```diff
@@ -0,0 +1,1229 @@
+            ),
+            CurvePlot(
+                title="Precision vs. Recall",
+                x_label="Recall",
+                y_label="Precision",
+                curves=[
+                    Curve(x=[0.0], y=[0.0], label="zeros with unmatched gt and unmatched inf"),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+        ),
+        (
+            "zeros with two matchings",
+            [
+                MulticlassInferenceMatches(
+                    matched=[],
+                    unmatched_gt=[
+                        (LabeledBoundingBox((3, 3), (4, 4), "b"), None),
+                    ],
+                    unmatched_inf=[],
+                ),
+                MulticlassInferenceMatches(
+                    matched=[],
+                    unmatched_gt=[],
+                    unmatched_inf=[
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "a", 0),
+                    ],
+                ),
+            ],
+            CurvePlot(
+                title="F1-Score vs. Confidence Threshold",
+                x_label="Confidence Threshold",
+                y_label="F1-Score",
+                curves=[
+                    Curve(x=[0.0], y=[0.0], label="zeros with two matchings"),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+            CurvePlot(
+                title="Precision vs. Recall",
+                x_label="Recall",
+                y_label="Precision",
+                curves=[
+                    Curve(x=[0.0], y=[0.0], label="zeros with two matchings"),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+        ),
+        (
+            "zeros, but one match for label a",
+            [
+                MulticlassInferenceMatches(
+                    matched=[
+                        (LabeledBoundingBox((3, 3), (4, 4), "a"), ScoredLabeledBoundingBox((3, 3), (4, 4), "a", 0)),
+                    ],
+                    unmatched_gt=[
+                        (LabeledBoundingBox((1, 1), (2, 2), "b"), None),
+                        (LabeledBoundingBox((1, 1), (2, 2), "a"), None),
+                    ],
+                    unmatched_inf=[],
+                ),
+            ],
+            CurvePlot(
+                title="F1-Score vs. Confidence Threshold",
+                x_label="Confidence Threshold",
+                y_label="F1-Score",
+                curves=[
+                    Curve(x=[0.0, 0.0], y=[0.0, 0.0], label="zeros, but one match for label a"),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+            CurvePlot(
+                title="Precision vs. Recall",
+                x_label="Recall",
+                y_label="Precision",
+                curves=[
+                    Curve(x=[0.0, 0.0], y=[0.0, 0.0], label="zeros, but one match for label a"),
```

Comment:
Hmm shouldn't precision be `1.0` at threshold of `0`?  I guess this will depend on whether to include or exclude the matched instance right @ the threshold.